Automattic/wp-calypso
# Add/hosting panel for non atomic

### Summary

This PR opens up the Hosting panel to be visible when a site is on the Business plan but isn't atomic. (Still encapsulated behind a development flag.)

It is important to remember that test data for the API is still included, so expect to see that during testing.

<hr>

### Screenshots

<img width="1051" alt="Screen Shot 2019-10-25 at 6 44 35 PM" src="https://user-images.githubusercontent.com/8892849/67608794-ce7c9f80-f757-11e9-8034-f35f5ee625be.png">

**Hosting panel when viewed for a site that is not atomic, but on a Business plan.**

<hr>

### Testing

* Spin up a local Calypso dev instance with this branch applied.
* Go to `http://calypso.localhost:3000/hosting/your.site`, where `your.site` is a site that is on the Business plan but not an Atomic site.
    * Notice that both cards are disabled and a banner is displayed, directing you to activate the Hosting panel features.
* Go to `http://calypso.localhost:3000/hosting/your.atomic.site`, ensuring that the site selected is an Atomic site.
    * Notice that the banner is _not_ displayed and the cards display associated test data. (Test data for PMA card should display button in 'loading' state.)
    * Ensure that the 'PHP & MySQL' option is displayed in the sidebar.

Fixes #36483 and #36484
